omero/developers/Web/Deployment.txt

```diff
@@ -68,10 +68,11 @@ Using WSGI (Unix/Linux)
 For convenience you may wish to run a web server under your local user account
 instead of using a system server for testing. Install Nginx and Gunicorn
 (See :doc:`/sysadmins/unix/install-web`) but generate a configuration file
-using the following command:
+using the following commands:
 
 ::
 
+    $ bin/omero config set omero.web.application_server 'wsgi-tcp'
```

Comment:
Yes, I think so, because we have no idea what the user's config will be when they try this.
And if we remove this, then users won't know what to set.